src/app/collective-rewards/leaderboard/BuildersLeaderBoardTable.tsx

```diff
@@ -60,14 +60,14 @@ export const BuildersLeaderBoardTable: FC = () => {
       rewardPercentage: (a: IRewardData, b: IRewardData) =>
         Number(a.rewardPercentage.current - b.rewardPercentage.current),
       lastCycleRewards: (a: IRewardData, b: IRewardData) => {
-        const aValue = a.lastCycleReward.rif.crypto.value + a.lastCycleReward.rbtc.crypto.value
-        const bValue = b.lastCycleReward.rif.crypto.value + b.lastCycleReward.rbtc.crypto.value
-        return aValue - bValue
+        const aValue = a.lastCycleReward.rif.amount.value + a.lastCycleReward.rbtc.amount.value
+        const bValue = b.lastCycleReward.rif.amount.value + b.lastCycleReward.rbtc.amount.value
+        return Number(aValue - bValue)
```

Comment:
Don't we risk to lose precision in this way?

Reply:
We may, let me change the bigint comparison.